[ENH] Some fixes, some enhancements and three new blocks (#129)
Geforce132/SecurityCraft: Changelog v1.8.x.md

- New: Reinforced Blue Ice (Thanks Redstone_Dubstep!)
- New: Reinforced Cobblestone Wall, Reinforced Mossy Cobblestone Wall
- New: Reinforced Blocks: Brick Wall, Prismarine Wall, Red Sandstone Wall, Mossy Stone Brick Wall, Granite Wall, Stone Brick Wall, Nether Brick Wall, Andesite Wall, Red Nether Brick Wall, Sandstone Wall, End Stone Brick Wall, Diorite Wall
- New: Reinforced Blocks: Observer, Buttons, Lever, Grass Block, Coarse Dirt, Podzol, Ice, Snow Block, Clay, Mycelium, Packed Ice, Nether Wart Block, Cobweb (Thanks Redstone_Dubstep!)
- New: Block mines for Coal Ore, Emerald Ore, Gold Ore, Iron Ore, Lapis Lazuli Ore, Nether Quartz Ore, and Redstone Ore
- New: Configuration option to turn off the darker textures of reinforced blocks
- New: Korean translation (Thanks mindy15963!)

> - New: When you press "Add", "Remove" or "Clear" in the Whitelist/Blacklist module screen, the textbox clears itself 

Reviewer: ```suggestion
- New: Pressing "Add", "Remove" or "Clear" in the Whitelist/Blacklist module screen clears the textbox
```

- New: The buttons in the Whitelist/Blacklist module screen activate and deactivate depending on the text field input and the amount of stored players

> - New: Sentries have a special HWYLA- and TOP-Tooltip that shows the owner, the current mode and the equipped modules

Reviewer: ```suggestion
- New: HWYLA and TOP now show a Sentry's owner, its current mode, and the equipped modules
```

> - New: Reinforced Doors, Fence gates and trapdoors have a greater range to detect nearby active reinforced blocks

Reviewer: This is a change. Move it to the bottom of the `Change` list.

- Change: Bouncing Betty now jumps as high as the player's head (Thanks Redstone_Dubstep!)
- Change: Zooming when viewing a Security Camera is no longer restriced to only three zoom levels
- Change: Creepers, Ocelots, and Endermen can now trigger Mines
- Change: Some reinforced block models are now randomly rotated (Thanks Redstone_Dubstep!)
- Change: White-/Blacklist Modules can now hold up to 50 players

> - Change: Sentries now attack Phantoms, Ghasts, Slimes, Magma Slimes, Shulkers and the Ender Dragon

Reviewer: Move down a line.
